<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTestinappi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Testinappi!</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTestiteksti">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Testitekstikentt√§!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Etunimi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
     </font>
    </property>
    <property name="text">
     <string>Sukunimi:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEtunimi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSukunimi">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTallenna">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tallenna</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTervehdys">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Moro!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAvaaSeuraava">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Siirry seuraavaan ikkunaan...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
